!!python/object:huggingface_hub.community.DiscussionWithDetails
author: dibo
conflicting_files: null
created_at: 2022-12-13 18:26:26+00:00
diff: null
endpoint: https://huggingface.co
events:
- !!python/object:huggingface_hub.community.DiscussionComment
  _event:
    author:
      avatarUrl: /avatars/b2742d4fae1a7117a74a51be28766a26.svg
      fullname: Chiara Di Bonaventura
      isHf: false
      isOrgMember: false
      isOwner: false
      isPro: false
      name: dibo
      type: user
    createdAt: '2022-12-13T18:26:26.000Z'
    data:
      edited: false
      editors:
      - dibo
      hidden: false
      isReport: false
      latest:
        author:
          avatarUrl: /avatars/b2742d4fae1a7117a74a51be28766a26.svg
          fullname: Chiara Di Bonaventura
          isHf: false
          isPro: false
          name: dibo
          type: user
        html: '<p>Hi! When I try to run the example, I get this error:<br><a rel="nofollow"
          href="https://cdn-uploads.huggingface.co/production/uploads/1670955680404-62de70fb86220b5cb895e199.png"><img
          alt="image.png" src="https://cdn-uploads.huggingface.co/production/uploads/1670955680404-62de70fb86220b5cb895e199.png"></a></p>

          <p>I think it''s linked to "tokenizer_file": null in tokenizer_config.json
          but not sure. Could you please have a look at it?</p>

          '
        raw: "Hi! When I try to run the example, I get this error: \r\n![image.png](https://cdn-uploads.huggingface.co/production/uploads/1670955680404-62de70fb86220b5cb895e199.png)\r\
          \n\r\nI think it's linked to \"tokenizer_file\": null in tokenizer_config.json\
          \ but not sure. Could you please have a look at it?"
        updatedAt: '2022-12-13T18:26:26.440Z'
      numEdits: 0
      reactions: []
    id: 6398c3d22da24ee95b526574
    type: comment
  author: dibo
  content: "Hi! When I try to run the example, I get this error: \r\n![image.png](https://cdn-uploads.huggingface.co/production/uploads/1670955680404-62de70fb86220b5cb895e199.png)\r\
    \n\r\nI think it's linked to \"tokenizer_file\": null in tokenizer_config.json\
    \ but not sure. Could you please have a look at it?"
  created_at: 2022-12-13 18:26:26+00:00
  edited: false
  hidden: false
  id: 6398c3d22da24ee95b526574
  type: comment
is_pull_request: false
merge_commit_oid: null
num: 1
repo_id: eetnawa/StereoKG-DT-UK
repo_type: model
status: open
target_branch: null
title: 'error with tokenizer '
